gneiss/plot/_regression_plot.py

```diff
@@ -0,0 +1,369 @@
+    p1.title.text = 'Explained Sum of Squares'
+    p1.title_location = 'above'
+    p1.title.align = 'center'
+    p1.title.text_font_size = '18pt'
+
+    # 2D scatter plot for prediction on PB
+    p2 = _projected_prediction(model, plot_width=w, plot_height=h)
+    p3 = _projected_residuals(model, plot_width=w, plot_height=h)
+    hm_p = _heatmap_summary(model.pvalues, model.coefficients())
+
+    # combine the cross validation, explained sum of squares tree and
+    # residual plots into a single plot
+    p = row(column(p2, p3), p1)
+    p = column(hm_p, p)
+    index_fp = os.path.join(output_dir, 'index.html')
+    with open(index_fp, 'w') as index_f:
+        index_f.write('<html><body>\n')
+        index_f.write('<h1>Simplicial Linear Regression Summary</h1>\n')
+        index_f.write(smry.as_html())
+        _deposit_results_html(index_f)
+
+        plot_html = file_html(p, CDN, 'Diagnostics')
+        index_f.write(plot_html)
+        index_f.write('</body></html>\n')
+
+
+# LME summary
+def lme_summary(output_dir: str, model: LMEModel, tree: TreeNode) -> None:
+    """ Summarizes the ordinary linear mixed effects model.
+
+    Parameters
+    ----------
+    output_dir : str
+        Directory where all of the regression results and
+        summaries will be stored.
+    model : LMEModel
+        Linear Mixed Effects model that contains the model fit and the
+        regression results.
+    tree : TreeNode
+        Tree object that defines the partitions of the features. Each of the
+        leaves correspond to the balances in the model.
+    """
+    # log likelihood
+    loglike = pd.Series({r.model.endog_names: r.model.loglike(r.params)
+                         for r in model.results})
+    w, h = 500, 300  # plot width and height
+    # Summary object
+    smry = model.summary()
+
+    t = _decorate_tree(tree, -loglike)
+    p1 = radialplot(t, edge_color='color', figsize=(800, 800))
+    p1.title.text = 'Loglikelihood of submodels'
+    p1.title_location = 'above'
+    p1.title.align = 'center'
+    p1.title.text_font_size = '18pt'
+
+    # 2D scatter plot for prediction on PB
+    p2 = _projected_prediction(model, plot_width=w, plot_height=h)
+    p3 = _projected_residuals(model, plot_width=w, plot_height=h)
+
+    hm_p = _heatmap_summary(model.pvalues, model.coefficients(),
+                            plot_width=900, plot_height=400)
+
+    # combine the cross validation, explained sum of squares tree and
+    # residual plots into a single plot
+    p = row(column(p2, p3), p1)
+    p = column(hm_p, p)
+
+    # Deposit all regression results
+    _deposit_results(model, output_dir)
+
+    index_fp = os.path.join(output_dir, 'index.html')
+    with open(index_fp, 'w') as index_f:
+        index_f.write('<html><body>\n')
+        index_f.write('<h1>Simplicial Linear Mixed Effects Summary</h1>\n')
+        index_f.write(smry.as_html())
+        _deposit_results_html(index_f)
```

Comment:
This is the second instance of _deposit_results_html and I'm sure you can see the similarities ...